safe-defaults/tmpfiles-enable.conf.in

```diff
@@ -6,7 +6,9 @@
 # if a symlinked config file in the ostree is no longer needed it should
 # be made empty rather than deleted, otherwise the change from L to r
 # in this file will be ineffective.
+L @SYSCONFDIR@/chromium-browser/policies/managed/01_doh.json - - - - @DATAROOTDIR@/eos-safe-defaults/chrome/policies/managed/01_doh.json
```

Comment:
`/etc/chromium-browser` is no longer used since Chromium became a flatpak. New files should not be placed there.

```suggestion
```

Reply:
The operative line is:

```
L /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@/1 - - - - @DATAROOTDIR@/eos-safe-defaults/chrome
```

which exposes the whole of `/usr/share/eos-safe-defaults/chrome/` to the Chromium Flatpak. I believe no adjustment is needed in this file to make this new file show up for Chromium.

Reply:
Ah!  Good point!
I tested it, and the result is same as you mentioned.